styles.css

```diff
@@ -0,0 +1,555 @@
+  width: 380px;
+  height: 210px;
+  padding: 0;
+  border: 0;
+  margin: 0;
+}
+
+.bra {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(249, 52, 52, 0.5),
+      rgba(249, 52, 52, 0.5)
+    ),
+    url(./img/img2.jpg);
+  background-repeat: no-repeat;
+  width: 380px;
+  height: 210px;
+  padding: 0;
+  border: 0;
+  margin: 0;
+}
+
+.cityName {
+  font-size: 34px;
+  font-weight: 900;
+}
+
+.cityNameText {
+  font-size: 12px;
+  font-weight: 900;
+}
+
+.backgroundGray {
+  background-color: #3c3f46;
+}
+
+.asa {
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.sponsor {
+  padding: 0 16px 16px;
+}
+
+ul {
+  padding: 0;
+}
+
+.partners {
+  padding-top: 24px;
+  padding-bottom: 12px;
+  font-weight: 900;
+}
+
+.grid2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid3 {
+  display: none;
+}
+
+.history {
+  background-color: white;
+}
+
+.menuOpen {
+  display: block;
+  position: fixed;
+  background-color: rgb(248, 248, 248);
+  top: 0;
+  left: 0;
+  margin: 0;
+  z-index: 1;
+  width: 100vw;
+  height: 100vh;
+  padding-left: 30px;
+  padding-top: 60px;
+  font-size: 32px;
+  line-height: 2;
+  text-align: center;
+}
+
+.closeButton {
+  text-align: end;
+  padding-right: 30px;
+}
+
+@media only screen and (min-width: 768px) {
+  header {
+    background-color: #3e3e3e;
+    margin: 0;
+    padding-right: calc((100% - 766px) / 2);
+  }
+
+  nav {
+    background-color: white;
+  }
+
+  .headerItems {
+    margin: 0;
+    justify-content: end;
+  }
+
+  .headerItems > * {
+    padding: 5px 10px;
+  }
+
+  .displayNoneSmall {
+    display: block;
+  }
+
+  .displayHideBig {
+    display: none;
+  }
+
+  .menuList {
+    padding-right: calc((100% - 766px) / 2);
+    display: flex;
+    align-items: center;
+  }
+
+  .menuList > * {
+    padding: 10px 20px;
+  }
+
+  .menuBorder {
+    border: #ec5242 solid 5px;
+  }
+
+  .navLogo {
+    width: 80px;
+    margin-left: calc((100% - 766px) / 2);
+  }
+
+  .mLeft20 {
+    margin-left: calc((100% - 766px) / 2);
+  }
+
+  .mRight20 {
+    margin-right: calc((100% - 766px) / 2);
+  }
+
+  .subText {
+    margin-right: calc((100% - 766px) / 2);
+    margin-left: calc((100% - 766px) / 2);
+  }
+
+  .squares {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    max-width: 768px;
+    margin-left: calc((100% - 766px) / 2);
+  }
+
+  .flexColumn {
+    flex-direction: column;
+  }
+
+  .programStyle {
+    justify-content: center;
+    margin: 3px;
+  }
+
+  .programStyle > * {
+    text-align: center;
+  }
+
+  .fs200 {
+    font-size: 120%;
+  }
+
+  .fs80 {
+    font-size: 80%;
+  }
+
+  .seeProgram {
+    margin-top: 30px;
+    background-color: #272a31;
+    padding: 16px;
+    text-decoration: underline;
+    border: none;
+  }
+
+  .playerSection {
+    display: grid;
+    grid-template-columns: 379px 379px;
+    margin-left: calc((100% - 766px) / 2);
+  }
```

Comment:
- [ ] Kindly give the featured section more spacing especially a `margin-top` and `margin-bottom` to your design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190196912-1cb632bd-a233-455b-b80f-122e72c4f548.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190197798-2fbe69e5-766a-45f5-bf4f-d082d4ce075f.png)